python/src/scipp/plot/plot3d.py

```diff
@@ -18,16 +18,21 @@ def plot3d(*args, filename=None, **kwargs):
     It is possible to add cut surfaces as cartesian, cylindrical or spherical
     planes.
     """
-
     sp = SciPlot3d(*args, **kwargs)
-
     if filename is not None:
         sp.savefig(filename)
-
     return sp
 
 
 class SciPlot3d(SciPlot):
+    """
+    Class for 3 dimensional plots.
+
+    It uses `pythreejs` to render the data as an interactive 3d point cloud.
+    All points are rendered as 2d squares for best performance.
+    It also holds a `panel3d` with additional control for spatial slicing based
```

Comment:
As noted elsewhere: This looks neither like a class name nor like an instance/member name?